felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -41,6 +41,7 @@ import (
 	apiv3 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
 	"github.com/projectcalico/api/pkg/lib/numorstring"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
```

Comment:
we usually import it this way

```suggestion
	log "github.com/sirupsen/logrus"
```

Reply:
There are other packages actually named `log`. It's convenient to use the actual package name for things like goimports automatically importing the right package as well.